Description: removing std
 Removal of std dependency which requires additional package
Author: Ed Neville <ed-debian@s5h.net>
Forwarded: not-needed
Last-Update: 2020-12-31
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: getopts/Cargo.toml
===================================================================
--- getopts.orig/Cargo.toml
+++ getopts/Cargo.toml
@@ -21,20 +21,8 @@ readme = "README.md"
 categories = ["command-line-interface"]
 license = "MIT/Apache-2.0"
 repository = "https://github.com/rust-lang/getopts"
-[dependencies.core]
-version = "1.0"
-optional = true
-package = "rustc-std-workspace-core"
-
-[dependencies.std]
-version = "1.0"
-optional = true
-package = "rustc-std-workspace-std"
 
 [dependencies.unicode-width]
 version = "0.1.8"
 [dev-dependencies.log]
 version = "0.4"
-
-[features]
-rustc-dep-of-std = ["unicode-width/rustc-dep-of-std", "std", "core"]
